be1-go/channel/coin/mod.go

```diff
@@ -1,4 +1,4 @@
-package digitalCash
+package coin
```

Comment:
You must be sure to stay consistent with the naming, across all projects, documentation, specification, messageData, and examples. Right now there is `cash`, `digitalCash`, `coin` and `transaction`. Please make sure only one is selected and used. 

Naming has a huge impact, much more than what we can think at first. If the channel is `coin`, then the messageData object should also be `coin`, not `transaction`. This is currently to case for all channels.